### DISCUSSION TITLE
Time series

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1957] [Q&A]
Hello.

I have 2 time series, one with geographic coverage by federal entities and the other with coverage by federal entity and municipalities.
In addition, with a breakdown by Sex (Total, Men, Women) and Populations (Speakers of an indigenous language, Disability, Afro-Mexican or Afro-descendant, Residents in another entity 5 years ago)

1. How can I make the map start in the second time series (coverage by state and municipalities), in the national coverage and breakdown of Sex - Total?

2. Why don't all the breakdowns appear on the map?
![image](https://user-images.githubusercontent.com/117297044/225989540-c69c1a1a-113a-4c89-88f6-3cfef1614646.png)

3. Can the Year column be used as a breakdown for specific indicators?

Thank you

**Comment by sarah-beasley** [comment]
To address your points:
1. You can do this by using the data_start_values field, which i believe you have already done now.
2. What other breakdowns are you expecting? This will be linked to your data, so if you include there, they should appear on your platform.
3. Year is the default breakdown on the x axis, what is special. You can use any arbitrary disaggregation you want, you would just need title it something other than 'year' in the column heading.
